### DISCUSSION TITLE
GHelper v0.30 - UI Concept

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#90] [Polls]
Hello fellow community,

I have a feeling that some people struggle to find Optimized mode from Armoury (in G-Helper it's just a checkbox under GPU modes) or Auto refresh rate setting. Also it feels that too much checkboxes make UI cluttered (it has too many features already).

So I had in mind to do a small revamp of an interface

Please feel free to download this build and test and **[share your opinion in this discussion](https://github.com/seerge/g-helper/discussions/91)**

Thanks.

**Comment by flemesre** [comment]
While I like both the checkbox and button looks, I feel like v0.30's UI loses information compared to the checkboxes in previous version. Without an explanation, "Optimised" mode and "Auto" refresh rate don't really explain what they do - for example, the first thought that comes to mind when I see "auto refresh rate" is that it's a content-adaptive refresh rate mode (as with basically every modern smartphones), rather than switching between modes when the laptop is plugged and unplugged.

I think the v0.30 UI looks nicer/cleaner, but is less functional and self-explanatory than previous versions.

**Reply by seerge** [reply]
I have added a small tooltips below buttons, that should restore explanations :) Just re-download same v0.30 build please

**Reply by ke1ne** [reply]
> I have added a small tooltips below buttons, that should restore explanations :) Just re-download same v0.30 build please

I would say the default Windows hint [design](https://learn.microsoft.com/en-us/windows/apps/design/controls/tooltips) fits more "predictable".

**Comment by MrxSiN** [comment]
As my laptop is G15 and a 2021 model, there will be no Ultimate Mode for gpu. On new UI, it leave a big gap between Standard and Optimized.

Also, im notice the icon and the highlight seem difference. Here the image, https://imgur.com/9IiUqpv

Suggestion, since the design using small tooltips below buttons, the OD word can be remove to make it look more clean. Extra can be changed to Hotkey since that is what the button currently do and maybe have a gap between Color button to state that its not just a hotkey for Aura.

**Reply by seerge** [reply]
I have fixed UI scaling and added proper handling situation without Ultimate mode. Can you check 
[GHelper.zip](https://github.com/seerge/g-helper/files/11008690/GHelper.zip)
 ?

**Comment by gloa7** [comment]
I love the new icons!

Any chance at a dark mode in the future?

**Reply by TomBonnot** [reply]
@astronaut701 is working on it, if you want to help there is a pull request available about it.

**Comment by LJGJM** [comment]
Nice GHelper UI, but just one problem, when your laptop doesnt support "ultimate" it will make a gap

**Reply by seerge** [reply]
Fixed gap 
[GHelper.zip](https://github.com/seerge/g-helper/files/11008690/GHelper.zip)

**Reply by LJGJM** [reply]
Thanks seerge

**Comment by restoration2699** [comment]
works on tuf fa70iu, but is it possible to return checkbox of turboboost on-off? it was very usefull by one-click enabling option.

**Reply by seerge** [reply]
That's a separate question :) But since turbo boost is now a dropdown (as bunch of people requested), i can't place it on main screen :) Also turbo boost adjustment becomes a bit obsolete when CPU PPT can be limited :)

**Reply by restoration2699** [reply]
unfortunately CPU PPT can't be adjusted on 2021 TUF

**Comment by TomBonnot** [comment]
Unpopular opinion but I think you are not helping our friend @astronaut701 haha
Tho it looks smooth and fine as I sad, it targets the soft mood : Useful, lightweight and powerful.

**Comment by Kulibchenko** [comment]
Is it possible to change the CPU Boost for each profile separately?

**Reply by seerge** [reply]
Nope, cause CPU Boost is part of windows power plan :)
